docs-src/modules/getting-started/pages/index.adoc

```diff
@@ -1,14 +1,146 @@
+
+[stream_viz,name=basic-sum]
+.Aggregating events as a timeline
+....
+[
+    {
+        "label": "Purchase.amount",
+        "kind": "discrete",
+        "data": [
+            [
+                {"t": 2, "v": "5"},
+                {"t": 5, "v": "2"},
+                {"t": 13, "v": "13"},
+                {"t": 18, "v": "4"}
+            ]
+        ]
+    },
+    {
+        "label": "Purchase.amount | sum()",
+        "kind": "continuous",
+        "data": [
+            [
+                {"t": 2, "v": "5"},
+                {"t": 5, "v": "7"},
+                {"t": 13, "v": "20"},
+                {"t": 18, "v": "24"}
+            ]
+        ]
+    }
+]
+....
+
+Timelines allow you to reason about temporal context, time travel, sequencing, time-series, and more. 
+They allow simple, composable, declarative queries over events. 
+Transforming and combining timelines allows you to intuitively express computations over events.
+
+****
+⭢ Read more about timelines in xref:overview:what-is-kaskada.adoc[]
+****
+
+With Kaskada, it's timelines all the way down - every operation's inputs and outputs are timelines. 
+The timeline is a flexible abstraction that can be used in different ways depending on your needs.
+In some cases, you may want to know what the timeline's value is at a specific point in time, in other cases you may want to know how the timeline's value changes over time.
+
+When making a query, you configure how to use the query's output timeline: you can use the timeline as either a _history_ or as a _snapshot_.
+
+* A timeline *History* contains a value each time the timeline changes, and each row describes a different entity at a different point in time. 
+* A timeline *Snapshot* contains a value for each entity at the _same_ point in time; each row is associated with a different entity, but all rows reflect the same point in time.
+
+This output may be written in different ways -- for example, they may be written to a Parquet file or sent as events to a stream.
+
+****
+⭢ Read more about converting to tables in xref:developing:queries.adoc#configuring-how-queries-are-computed[Queries]
+****
+
+== Kaskada's basic architecture
+
+Kaskada is implemented as a standalone service with two main components.
+
+* The *engine* is a stateless compute engine for transforming event data.
+* The *manager* is an (optionally) stateful API for communicating with clients.
+
+These components are distributed as standalone, precompiled binaries.
+The Kaskada service is accessed using a gRPC API, and multiple clients exist.
+You can access the API using the provided Python client or using the CLI binary.
+Kaskada can be configured to run as a remote service or as a local process.
+
+****
+⭢ Read more about installing Kaskada in xref:ROOT:installing.adoc[]
+****
+
+== Kaskada's data model
+
+Timelines begin with xref:developing:tables.adoc[tables].
+Tables are how Kaskada stores input events.
```

Comment:
`stores` vs. `retrieves` or `accesses`? specifically, with pulsar tables it's not just storage but retrieval, right?